.github/workflows/main.tf

```diff
@@ -48,6 +48,13 @@ resource "azurerm_mssql_server" "musicfestival" {
   administrator_login_password = "P@ssw0rd1234"
```

Comment:
since it's public repo, let's remove sensitive info. Also, please change it to sth else since it's already revealed on the PR.